editor/components/post-type-support-check/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { PostTypeSupportCheck } from '../index';
+
+describe( 'PostTypeSupportCheck', () => {
+	it( 'should not render if there\'s no support check provided', () => {
+		const wrapper = shallow( <PostTypeSupportCheck>foobar</PostTypeSupportCheck> );
+		expect( wrapper.type() ).toBe( null );
+	} );
+
+	it( 'should render if both thumbnail and post-thumbnails are supported', () => {
+		const themeSupports = {
+			'post-thumbnails': true,
+		};
+		const postType = {
+			supports: {
+				thumbnail: true,
+			},
+		};
+		const supportKeys = 'thumbnail';
+		const wrapper = shallow( <PostTypeSupportCheck
+			supportKeys={ supportKeys }
+			postType={ postType }
+			themeSupports={ themeSupports }>foobar</PostTypeSupportCheck> );
+		expect( wrapper.type() ).not.toBe( null );
```

Comment:
Can we use more targeted expectation in here to ensure it properly renders `foobar` text? One of the ways of doing it is to use [toHaveText](https://github.com/FormidableLabs/enzyme-matchers#tohavetext) matcher.